server/mlabns/handlers/lookup.py

```diff
@@ -23,7 +23,7 @@ def _create_tool_url(fqdn, http_port):
     Returns:
         A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (fqdn, http_port)
+    return str('http://%s:%s' % (fqdn, http_port))
```

Comment:
Yeah, just looking at it I agree it looks redundant. If I take it out though, I get the unicode error we see on mlab-ns now (http://mlab-ns.appspot.com/npad?format=redirect). So I definitely have to cast something as a string, but I'm open to more clever/logical approaches.